xrdp/xrdp_mm.c

```diff
@@ -991,11 +991,185 @@ xrdp_mm_process_rail_drawing_orders(struct xrdp_mm *self, struct stream *s)
     return rv;
 }
 
+/******************************************************************************/
+static int
+dynamic_monitor_open_response(intptr_t id, int chan_id, int creation_status)
+{
+    struct xrdp_process *pro;
+    struct xrdp_wm *wm;
+    struct stream *s;
+    int bytes;
+
+    LOG(LOG_LEVEL_INFO, "dynamic_monitor_open_response: chan_id %d creation_status 0x%8.8x", chan_id, creation_status);
+    if (creation_status != 0)
+    {
+        LOG(LOG_LEVEL_INFO, "dynamic_monitor_open_response: error");
```

Comment:
Happy for a LOG() here as this is a user error, but shouldn't it be LOG_LEVEL_ERROR ?

Reply:
Updated.